### DISCUSSION TITLE
Pictures name

### DISCUSSION THREAD

**Discussion by falbrieu** [opengisch/QField#4335] [Q&A]
When I take pictures on field, they are saved with the date and time. Is there any way I can choose the name that pictures are saved with?

**Comment by lp-dj** [comment]
https://docs.qfield.org/how-to/pictures/#configurable-attachment-path

Is there also a way to do this setting if you don't use the QFieldSync plugin?

**Reply by falbrieu** [reply]
Amazing, that worked. Thanks so much @lp-dj !

**Reply by lp-dj** [reply]
Glad if I could help, @falbrieu.

But thanks not to me but to those who implemented the function and documented it so nicely ;)

A good way to show appreciation is to actually read the documentation ;)

But no offense: I like to help and have overlooked things before too.

Greetings and happy (Q)fieldwork
